### ISSUE TITLE
Cannot read property 'FIT' of undefined

### ISSUE THREAD

**Issue by dyspop** [rexrainbow/phaser3-rex-notes#129]
I'm assuming this isn't installed properly. I appreciate your plugins but I'm a phase noob and the documentation is a little confusing especially around usage and installation.

Here's the source (mostly copied from your example):

```
<!doctype html> 
<html lang="en"> 
<head> 
    <meta charset="UTF-8" />
    <title>Making your first Phaser 3 Game - Part 7</title>
    <script src="//cdn.jsdelivr.net/npm/phaser@3.11.0/dist/phaser.js"></script>
    <style type="text/css">
        html, body {
          height: 100%;
        }

        body {
          margin: 0;
          padding: 0;
          background: #222;
          color: #eee;
          font: caption;
        }
    </style>
</head>
<body>

<script type="text/javascript">
const Random = Phaser.Math.Between;

class Demo extends Phaser.Scene {
    constructor() {
        super({
            key: 'examples'
        })
    }

    preload() {}
    create() {
        var print = this.add.text(0, 0, 'Click any tile');

        var staggeraxis = 'y';
        var staggerindex = 'odd';
        var board = this.rexBoard.add.board({
            grid: {
                gridType: 'hexagonGrid',
                x: 60,
                y: 60,
                size: 30,
                staggeraxis: staggeraxis,
                staggerindex: staggerindex
            }
        })
            .setInteractive()
            .on('tiledown', function (pointer, tileXY) {
                print.text = `${tileXY.x},${tileXY.y}`;
            })

        var tileXYArray = board.fit(this.rexBoard.hexagonMap.hexagon(board, 4));

        var graphics = this.add.graphics({
            lineStyle: {
                width: 1,
                color: 0xffffff,
                alpha: 1
            }
        });
        var tileXY, worldXY;
        for (var i in tileXYArray) {
            tileXY = tileXYArray[i];
            graphics.strokePoints(board.getGridPoints(tileXY.x, tileXY.y, true), true);

            worldXY = board.tileXYToWorldXY(tileXY.x, tileXY.y);
            this.add.text(worldXY.x, worldXY.y, `${tileXY.x},${tileXY.y}`).setOrigin(0.5);
        }
    }

    update() { }
}

var config = {
    type: Phaser.AUTO,
    parent: 'phaser-example',
    width: 800,
    height: 600,
    scale: {
        mode: Phaser.Scale.FIT,
        autoCenter: Phaser.Scale.CENTER_BOTH,
    },
    scene: Demo,
    plugins: {
          scene: [{
              key: 'rexBoard',
              plugin: rexboardplugin,
              mapping: 'rexBoard'
          }]
    },
    debug: true
};

var game = new Phaser.Game(config);

</script>
<footer><div id="phaser-example"></div></footer>
</body>
</html>
```

error is line 82:
`         mode: Phaser.Scale.FIT,`

**Comment by rexrainbow** [comment]
3.11.0 might not have Scale feature yet, please use latest stable released version : [3.24.1](https://github.com/photonstorm/phaser/releases/tag/v3.24.1)

**Comment by dyspop** [comment]
That cleared the error.
